homeassistant/components/sensor/wunderground.py

```diff
@@ -105,8 +115,31 @@ def state(self):
                 return int(self.rest.data[self._condition][:-1])
             else:
                 return self.rest.data[self._condition]
-        else:
-            return STATE_UNKNOWN
+
+        if self._condition == 'alerts':
+            if self.rest.alerts is None:
+                self.update()
```

Comment:
You're never allowed to call an `update` method inside a property. Just return STATE_UNKNOWN if `alerts` is None. `update` has already been called and it couldn't fix it.